lightbulb/decorators.py

```diff
@@ -75,7 +75,7 @@ def decorate(func: CommandCallbackT) -> CommandCallbackT:
     return decorate
 
 
-def command(
+def command(  # noqa: D417
```

Comment:
Could you copy the description from `lightbulb/app.py` to each time this is used?